platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -11,113 +11,162 @@ For details, visit [Azure Free Account][az-create-account].
+1. Go to your new Batch account, then select **Access Keys**.
+1. Store the access keys for your Azure Batch account, to be used when you create a Seqera compute environment.
     :::caution
     A newly-created Azure Batch account may not be entitled to create virtual machines without making a service request to Azure.
     See [Azure Batch service quotas and limits][az-batch-quotas] for more information.
     :::
-
-14. Select the **+ Quotas** tab of the Azure Batch account to check and increase existing quotas if necessary.
-15. Select **+ Request quota increase** and add the quantity of resources you require. Here is a brief guideline:
-
+1. Select the **+ Quotas** tab of the Azure Batch account to check and increase existing quotas if necessary.
+1. Select **+ Request quota increase** and add the quantity of resources you require. Here is a brief guideline:
     - **Active jobs and schedules**: Each Nextflow process will require an active Azure Batch job per pipeline while running, so increase this number to a high level. See [here][az-learn-jobs] to learn more about jobs in Azure Batch.
     - **Pools**: Each platform compute environment requires one Azure Batch pool. Each pool is composed of multiple machines of one virtual machine size.
-
     :::note
     To use separate pools for head and compute nodes, see [this FAQ entry](../faqs.mdx#azure).
     :::
-
     - **Batch accounts per region per subscription**: Set this to the number of Azure Batch accounts per region per subscription. Only one is required.
     - **Spot/low-priority vCPUs**: Platform does not support spot or low-priority machines when using Forge, so when using Forge this number can be zero. When manually setting up a pool, select an appropriate number of concurrent vCPUs here.
     - **Total Dedicated vCPUs per VM series**: See the Azure documentation for [virtual machine sizes][az-vm-sizes] to help determine the machine size you need. We recommend the latest version of the ED series available in your region as a cost-effective and appropriately-sized machine for running Nextflow. However, you will need to select alternative machine series that have additional requirements, such as those with additional GPUs or faster storage. Increase the quota by the number of required concurrent CPUs. In Azure, machines are charged per cpu minute so there is no additional cost for a higher number.
 
-### Compute environment
+### Credentials
+
+There are two types of Azure credentials available: access keys and service principals. Access keys are simple to use but provide full access to the Storage and Batch accounts. Azure allows only two access keys per account, making them a single point of failure. A service principal provides an account which can be granted access to Azure Batch and Storage resources. Service principals therefore enable role-based access control with more precise permissions. Moreover, some Azure Batch features are only available when using a service principal.
```

Comment:
I found this hard to follow - suggested change attached
```suggestion
There are two types of Azure credentials: access keys and service principals. Access keys are simple to use and provide full access to Azure Storage and Batch accounts. Azure allows only two access keys per account, making them a single point of failure. Service principals require more configuration as access to Azure Batch and Storage resources must be granted. Service principals therefore enable role-based access control with more precise permissions. Moreover, some Azure Batch features are only available when using a service principal.
```

Reply:
Reformatting as ordered lists.